cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -864,6 +864,17 @@ sub external_services {
         };
     }
 
+    if ( my $livejournal = $u->prop( 'livejournal' ) ) {
+        my $elivejournal = LJ::eurl( $livejournal );
```

Comment:
This should be `my $livejournal = ...` so that we use the escaped version in the following `push`.